### PR TITLE
Move to Netlify

### PR DESCRIPTION
**Pull request by danfishgold**
As discussed in #1 we can't really use Glitch, because the server goes to sleep very quickly and it requires a long wait every time it's queried.

This PR changes the code from an Express server to a Netlify static site (which is currently just a single page) with a serverless function that generates the pdfs.

### CODE REVIEW

File: functions/make-pdf/make-pdf.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const { generatePdfBuffer } = require('./pdf-generation')
+
+async function handler(event) {
+  try {
+    const path = event.queryStringParameters.path
+    console.log(path)
```
Comment (katjam) [inline functions/make-pdf/make-pdf.js]: Do you want to leave this in?

**Review by katjam — APPROVED** [review]
Apart from that one possibly unintentional console log - this looks amazing! Sorry it took me so long to get around to review.

File: functions/make-pdf/make-pdf.js (hunk 1 above)
Comment (danfishgold) [reply functions/make-pdf/make-pdf.js]: oops! nice catch

**Comment by katjam** [comment]
@danfishgold Have you deployed this to a production netlify somewhere? I'd like to link out to it from the www site. Maybe add the url to the README here?

**Comment by danfishgold** [comment]
I just set up #3 with a bunch of changes so that this runs on netlify (and not just locally). there's a nice long description of the changes, but the actual code change is pretty minimal

**Pull request merged**